CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.5.0
+
+This is a feature release that also contains bug fixes.
+
+* Added a new "v2" API that is based on the [Jookia](https://jolokia.org) library. Jookia
```

Comment:
I don't think we need to deprecate it yet.  There are a couple of typos above, though, `Jolokia` vs. `Jookia`.